src/components/account-menu/_account-menu.scss

```diff
@@ -68,12 +68,13 @@ $hmrc-transition-type: "all";
 }
 
 .hmrc-account-menu__link {
+  @include govuk-typography-responsive($size: 16);
```

Comment:
interesting that it's had the impact (if I'm not reading screenshots incorrectly) of making fonts smaller on mobile - I guess that's intended but might be one to confirm is way it should be with DR I think @Jon-Rowe-HMRC / @timsb